action/repo_update.go

```diff
@@ -124,6 +130,8 @@ EXAMPLES:
     $ {{.HelpName}} --org MyOrg --repo MyRepo --timeout 90
   4. Update a repository when config or environment variables are set.
     $ {{.HelpName}}
+	5. Update a repository with a new build number.
+		$ {{.HelpName}} --org MyOrg --repo MyRepo --counter 200
```

Comment:
Could we get the formatting of this to be in-line with the other examples?

Reply:
RIP VSCode lied to me again. Will update.

Reply:
Should fix it https://github.com/go-vela/cli/pull/234/commits/a212a2d3a0865ceb1379436bec010e9d4f385e4d